appveyor.yml

```diff
@@ -0,0 +1,43 @@
+os: unstable
+environment:
+  matrix:
+  - nodejs_version: "6"
+  - nodejs_version: "7"
+
+platform:
+  - x86
```

Comment:
Same here, 32bit windows is rare these days and the odds it works on one arch and not the other is quite slim.  This just adds build time.